contracts/crypto-verify/Cargo.toml

```diff
@@ -41,5 +42,7 @@ sha2 = "0.10"
 sha3 = "0.10"
 
 [dev-dependencies]
+ark-ec = "0.4.2"
+ark-serialize = { version = "0.4.2", default-features = false }
```

Comment:
Do I understand correctly that we can remove all of those dependencies in the contract the moment we expose a serialized G1 generator and a serialized G2 generator constant from cosmwasm-std? If so, let's do that

Reply:
Yep. I will add two constants for the compressed elements (and some tests to verify they are correct).